cmd/mdatagen/loader.go

```diff
@@ -52,15 +52,13 @@ func (mn attributeName) RenderUnexported() (string, error) {
 
 // ValueType defines an attribute value type.
 type ValueType struct {
-	// ValueType is type of the metric number, options are "double", "int".
-	ValueType pcommon.ValueType
+	// ValueType is type of the attribute value.
+	ValueType pcommon.ValueType `validate:"required,notblank"`
```

Comment:
Is this only called on `go generate`?

I have a vague memory that `github.com/go-playground/validator` is a rather heavy and caused a significant slowdown in the last app I've used it in. Mind you, this was some time ago so this may not be valid.

Reply:
The dependency is only being used by `cmd/mdatagen` which is called by `go generate`. It's actually already in use, so I think this is fine either way.

Reply:
Yes, it's been there for a while. I believe we should still migrate to custom `Validate()` methods for consitency